src/hermes/commands/deposit/invenio.py

```diff
@@ -158,13 +169,14 @@ def deposit(click_ctx: click.Context, ctx: CodeMetaContext):
     response = session.post(publish_url)
     if not response.ok:
         _log.error(f"Could not publish deposit via {publish_url!r}")
+        _log.debug(response.text)
         click_ctx.exit(1)
 
     record = response.json()
     _log.info("Published record: %s", record["links"]["record_html"])
 
 
-def _resolve_latest_invenio_id(ctx: CodeMetaContext) -> str:
+def _resolve_latest_invenio_id(ctx: CodeMetaContext) -> (str, dict):
```

Comment:
```suggestion
def _resolve_latest_invenio_id(ctx: CodeMetaContext) -> Tuple[str, dict]:
```

From Pylance:

>Tuple expression not allowed in type annotation
>  Use Tuple[T1, ..., Tn] to indicate a tuple type or Union[T1, T2] to indicate a union type

(needs `from typing import Tuple`)